spotify_player/src/event/popup.rs

```diff
@@ -259,6 +267,100 @@ fn handle_command_for_queue_popup(
     Ok(true)
 }
 
+fn handle_key_sequence_for_create_playlist_popup(
+    key_sequence: &KeySequence,
+    client_pub: &flume::Sender<ClientRequest>,
+    state: &SharedState,
+) -> Result<bool> {
+    {
+        let mut ui = state.ui.lock();
+        let (name, desc, current_field) = match ui.popup {
+            Some(PopupState::PlaylistCreate {
+                ref mut name,
+                ref mut desc,
+                ref mut current_field,
+            }) => (name, desc, current_field),
+            _ => return Ok(false),
+        };
+        if key_sequence.keys.len() == 1 {
+            if let Key::None(c) = key_sequence.keys[0] {
+                match c {
+                    crossterm::event::KeyCode::Char(c) => {
+                        match &current_field {
+                            PlaylistCreateCurrentField::Name => {
+                                name.push(c);
+                            }
+                            PlaylistCreateCurrentField::Desc => {
+                                desc.push(c);
+                            }
+                        }
+                        return Ok(true);
+                    }
+                    crossterm::event::KeyCode::Backspace => {
+                        match &current_field {
+                            PlaylistCreateCurrentField::Name => {
+                                if !name.is_empty() {
+                                    name.pop().unwrap();
+                                }
+                            }
+                            PlaylistCreateCurrentField::Desc => {
+                                if !desc.is_empty() {
+                                    desc.pop().unwrap();
+                                }
+                            }
+                        }
+                        return Ok(true);
+                    }
+                    crossterm::event::KeyCode::Tab => {
```

Comment:
```suggestion
                    crossterm::event::KeyCode::Tab | crossterm::event::KeyCode::BackTab => {
```

can also handle shift+Tab